### DISCUSSION TITLE
Diplome - DL Gridmaster falscher Text

### DISCUSSION THREAD

**Discussion by antonov124** [wavelog/wavelog#942] [Q&A] [answered]
Unter den DL Gridmaster Diplome, ist der US Text wenn man auf Diplominfo drückt ...
`US Gridmaster Diplom
Der GridMaster Award ist die renommierteste AMSAT-Auszeichnung, die erstmals 2014 von der Star Comm Group eingeführt wurde. Sie steht allen Funkamateuren weltweit zur Verfügung, die es schaffen, alle 488 Planquadrate in den USA via Satellit zu arbeiten und für jeden Kontakt QSL-Bestätigungen vorlegen können.

Offizielle Informationen der [Webseite](https://www.amsat.org/gridmaster/): Die Zwei-Wege-Kommunikation muss von jedem Planquadrat aus mit einem Amateurfunk-Satelliten erfolgt sein. Es ist kein minimaler Signalbericht erforderlich. Kontakte müssen von der gleichen Stelle oder von Orten gemacht werden, von denen keine zwei mehr als 200 Kilometer voneinander entfernt sind. Die Bestätigung des Bewerbers in der Auszeichnungs-Beantragung dient als Bestätigung, dass er sich an die Abstandsregel hält. Personen können mehrere GridMaster Auszeichnungen beantragen und erhalten, wenn sie von einem anderen Ort erreicht werden, der in einem anderen 200-Kilometer-Kreis ist.

Diese Karte zeigt nur QSOs, die über SAT gearbeitet wurden.

`

**Comment by phl0** [comment, marked as answer]
Das ist der Fall, weil es keine anderen / offiziellen Gridmaster gibt. Daher ist der Text der gleiche, der Award basiert aber eben auf den Grids des jeweiligen Landes. Same for HB9, LX, ...

**Comment by antonov124** [comment]
Guten morgen danke für die Information ;)
